### PR TITLE
Requiem

### PR DESCRIPTION
**Pull request by KnowOne134**
assault Reguiem, had a problem with pets from drgoons so added spawn point for them to be at instead of like how reg pets work and the mob spawns the pet when it engages and follows

### CODE REVIEW

File: scripts/zones/Periqia/npcs/Rune_of_Release.lua [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    return 1
+
+end
+
+function onEventUpdate(player,csid,option)
+end
+
+function onEventFinish(player,csid,option)
+
+	local instance = player:getInstance()
```
Comment (nfaletra) [inline scripts/zones/Periqia/npcs/Rune_of_Release.lua]: Tab -> Spaces

File: scripts/zones/Periqia/npcs/Rune_of_Release.lua [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    local chars = instance:getChars()
+    local id = instance:getID()
+    local points = 0
+	local playerpoints = ((#chars -3)*100)
```
Comment (nfaletra) [inline scripts/zones/Periqia/npcs/Rune_of_Release.lua]: Another Tab

File: sql/instance_list.sql [hunk 3]
```diff
@@ -77,7 +77,7 @@ INSERT INTO `instance_list` VALUES (0,'TEST',0,0,0.000,0.000,0.000,0,-1,-1,-1,-1
 -- INSERT INTO `instance_list` VALUES (29,'operation:black_pearl',61,30,0.000,0.000,0.000,0,-1,-1,-1,-1);
 -- INSERT INTO `instance_list` VALUES (30,'better_than_one',61,30,0.000,0.000,0.000,0,-1,-1,-1,-1);
 -- INSERT INTO `instance_list` VALUES (31,'seagull_grounded',79,30,-350,-15.245,380,0,-1,-1,-1,-1);
--- INSERT INTO `instance_list` VALUES (32,'requiem',79,30,0.000,0.000,0.000,0,-1,-1,-1,-1);
+-- INSERT INTO `instance_list` VALUES (32,'requiem',79,30,-458.409,-10.000,-320.966,90,-1,-1,-1,-1);
```
Comment (nfaletra) [inline sql/instance_list.sql]: This line is still commented out. Is that intentional?

Comment (KnowOne134) [reply sql/instance_list.sql]: Im not sure if the team wants them open until all complete. The npcs the start the missions are also commented out so you can even pick the mission unless you remove the --

Comment (KnowOne134) [reply sql/instance_list.sql]: if we want to open im game with removing the -- think this is the 5th or 6th assault ive made so far. There is at least 1 per zone working

**Comment by KnowOne134** [comment]
Conflicts resolved

**Pull request merged**